internal/api/js/chrome/chrome.go

```diff
@@ -61,68 +52,30 @@ func MustRunAsyncFn[T any](t ct.TestLike, ctx context.Context, js string) *T {
 	return result
 }
 
-func RunHeadless(onConsoleLog func(s string), requiresPersistance bool, listenPort int) (*Browser, error) {
+func RunHeadless(onConsoleLog func(s string), listenPort int) (*Tab, error) {
+	// make a Chrome browser
 	browser, err := GlobalBrowser()
 	if err != nil {
-		return nil, err
+		return nil, fmt.Errorf("GlobalBrowser: %s", err)
 	}
```

Comment:
To confirm: these changes are nothing to do with the subject of the commit, right?